frontend/src/router/pages/EditAndReview.jsx

```diff
@@ -1,7 +1,73 @@
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { faMagnifyingGlass } from "@fa-kit/icons/classic/solid";
+import { useApiGet } from "@/hooks/useApi";
+import EditAndReviewTable from "@/components/EditAndReviewTable";
+import { useState } from "react";
+import orderBy from "lodash/orderBy";
 
 function EditAndReview() {
+  const { data, loading } = useApiGet("/parks");
+
+  // table filter state
+  const [nameFilter, setNameFilter] = useState("");
+  const [statusFilter, setStatusFilter] = useState("");
+
+  // table sorting state
+  const [sortColumn, setSortColumn] = useState("parkName");
+  const [sortOrder, setSortOrder] = useState("asc");
+
+  function resetFilters() {
+    setNameFilter("");
+    setStatusFilter("");
+  }
+
+  function updateSort(column, order) {
+    setSortColumn(column);
+    setSortOrder(order);
+  }
+
+  // add status values to parks
+  const parks =
+    data?.map((park) => ({
+      ...park,
+      status: `TODO: Active ${park.id}`,
+    })) || [];
+
+  // unique values for filter dropdowns
+  const statusValues =
+    parks
+      .map((park) => park.status)
+      .filter((value, index, self) => self.indexOf(value) === index) || [];
```

Comment:
Can we try using a Set? I think it'd be a bit more efficient.

Reply:
I just changed this to a hardcoded list of all of the status values in the end. That previous code was more useful for when the values were unknown